Description: adjust cabal dependencies
 Why so tight dependencies in the cabal file? These other versions have
 been extensively tested and also work ok. I filed a bug report upstream
 about this earlier.

--- haskell-github-0.2.1.orig/github.cabal
+++ haskell-github-0.2.1/github.cabal
@@ -143,14 +143,14 @@ Library
   Build-depends: base >= 4.0 && < 5.0,
                  time,
-                 aeson == 0.6.0.0,
+                 aeson == 0.6.0.1,
-                 attoparsec == 0.10.1.0,
+                 attoparsec == 0.10.1.1,
                  bytestring,
                  containers,
                  text,
                  old-locale,
                  HTTP,
                  network,
-                 http-conduit == 1.2.6,
+                 http-conduit == 1.2.5,
                  uri,
                  failure,
                  http-types,
